xml/System.Net.Mail/SmtpClient.xml

```diff
@@ -1737,14 +1746,33 @@ The classes shown in the following table are used to construct email messages th
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="4" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="from">To be added.</param>
-        <param name="recipients">To be added.</param>
-        <param name="subject">To be added.</param>
-        <param name="body">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="from">A <see cref="T:System.String" /> that contains the address information of the message sender.</param>
+        <param name="recipients">A <see cref="T:System.String" /> that contains the addresses that the message is sent to.</param>
```

Comment:
@aik-jahoda should we mention what separator character to use when the string contains more than one recipient?

```suggestion
        <param name="recipients">The addresses that the message is sent to.</param>
```